spinalcordtoolbox/deepseg/models.py

```diff
@@ -236,6 +286,28 @@ def display_list_tasks():
     exit(0)
 
 
+def display_long_description_task(name_task):
+    value = sct.deepseg.models.list_tasks()[name_task]
+
+    indent_len = len("LONG_DESCRIPTION: ")
+    print("{}{}".format("TASK:".ljust(indent_len), name_task))
+    print('\n'.join(textwrap.wrap(value['long_description'],
+                    width=shutil.get_terminal_size()[0],
+                    initial_indent="LONG_DESCRIPTION: ",
+                    subsequent_indent=' '*indent_len)))
+    print("{}{}".format("URL:".ljust(indent_len), value['url']))
+
+    path_models = [sct.deepseg.models.folder(name_model)
+                   for name_model in value['models']]
+    if all([sct.deepseg.models.is_valid(path_model) for path_model in path_models]):
+        installed = colored.stylize("Yes", colored.fg('green'))
+    else:
+        installed = colored.stylize("No", colored.fg('red'))
+    print("{}{}".format("INSTALLED:".ljust(indent_len), installed))
```

Comment:
NB: If we like the changes from https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/378, then we would need to rework this to no longer use `colored.stylize`, as was done in [d780a61](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3782/commits/d780a617b2bd9872ae8ba841b0fd18f451aa3dea).